TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/UsbChangesListenerIntentHandler.java

```diff
@@ -10,34 +10,34 @@
 
 import static com.novoda.tpbot.bot.AndroidMovementService.ACTION_USB_PERMISSION;
 
-class MovementServiceIntentHandler {
+class UsbChangesListenerIntentHandler {
```

Comment:
Just handles intents and forwards to the correct listener method.

Reply:
I really like this method of handling intents, single responsibility with no other code clouding everything up.